src/poggit/release/submit/SubmitFormAjax.php

```diff
@@ -650,7 +658,7 @@ private function getFields() {
 corresponding contributors and translators too).
 EOD
             ,
-            "refDefault" => $this->refRelease->authors,
+            "refDefault" => $useRefAuthors ? $this->refRelease->authors : null,
```

Comment:
To help with BC, use new list of authors to include the owner if previous release didn't have them listed.